### ISSUE TITLE
Do we have a plan to support multiple field values?

### ISSUE THREAD

**Issue by qudongfang** [OpenObservability/OpenMetrics#139]
Do we have a plan to support multiple field values?

Some metrics are produced together and will be used in a whole. Such as the metrics describing java virtual machine memory management.

Influxdb has its support.

1.  <https://docs.influxdata.com/influxdb/v1.7/concepts/glossary/#field-value>
1. <https://docs.influxdata.com/influxdb/v1.7/write_protocols/line_protocol_reference/>

**Comment by SuperQ** [comment]
No, even when samples are meant to be used together, they are still considered separate metrics.

For example, Historams follow this pattern.

It's possible in a future version to provide this as a wire transport optimization, but not at this time.

**Comment by madan** [comment]
@SuperQ Can we go about closing this.
